packages/fxa-payments-server/src/routes/Product/en.ftl

```diff
@@ -0,0 +1,2 @@
+product-invoice-preview-error-title = Problem loading invoice preview
+product-invoice-preview-error-text = product-invoice-preview-error-text
```

Comment:
The English string for `product-invoice-preview-error-text` here is missing and just repeating the ID.

Reply:
Whoops! C&P error. Thanks for spotting.